istore--0.1.1--0.1.2.sql

```diff
@@ -1,38 +1,107 @@
 ----functions----
+CREATE FUNCTION istore_agg_finalfn_pairs(internal)
+ RETURNS istore
+ LANGUAGE c
+ IMMUTABLE STRICT
+AS 'istore', $function$istore_agg_finalfn_pairs$function$;
+
+CREATE OR REPLACE FUNCTION bigistore_agg_finalfn(internal)
+ RETURNS bigistore
+ LANGUAGE c
+ IMMUTABLE STRICT
+AS 'istore', $function$bigistore_agg_finalfn_pairs$function$;
+
+CREATE FUNCTION istore_max_transfn(internal, bigistore)
+ RETURNS internal
+ LANGUAGE c
+ IMMUTABLE
+AS 'istore', $function$bigistore_max_transfn$function$;
+
+----
+CREATE FUNCTION istore_min_transfn(internal, bigistore)
+ RETURNS internal
+ LANGUAGE c
+ IMMUTABLE
```

Comment:
btw shouldn't the aggregate functions be declared `STRICT`. This would end the `PG_ARGISNULL(1)` calls below.
Speaking of which, there are other places with `PG_ARGISNULL` calls that appear redundant to me (i.e. `istore_array_add` to name one such function – this is not part of the PR though, don't get me wrong!).